_pages/algorithmia-enterprise/advanced-reporting.md

```diff
@@ -1,7 +1,8 @@
 ---
+exclude_from_search: true
 layout: article
 title:  "Advanced Reporting"
-permalink: algorithmia-enterprise/advaned-reporting/
+permalink: algorithmia-enterprise/advanced-reporting/
 excerpt: "Gain insight into platform usage and enhance ML governance with Algorithmia's advanced reporting capabilities"
 excerpt-short: "Algorithmia advanced reporting for governance"
 categories: [algorithmia-enterprise]
```

Comment:
agree 💯 . noticed that things were all out of order and wanted to talk about what the best order is, because i was thinking maybe title at the top, etc. but i'm fine with just alphabetical. i was thinking i could do the ordering changes all at once instead of also changing things that affect functionality at the same time. I'll do that soon.

Reply:
no hurry, but just wanted to mention. Thanks.